packages/presentation/src/components/ObjectPopup.svelte

```diff
@@ -58,11 +58,12 @@
 
   export let create: ObjectCreate | undefined = undefined
   export let readonly = false
+  export let preselect: ((itemId: Ref<Doc>, select: () => void) => void) | undefined = undefined
```

Comment:
Why we need it? It looks very strange, please use selected/selectedObjects or do a custom component.

Reply:
We need it to perform some checks before we actually mark or unmark an item. In my example I need to show a confirmation dialog and only mark the clicked item if the user confirmed the action in the dialog. 

It's not possible to use `selected/selectedObjects` for that, cause when we open a popup with `showPopup` we just pass the initial props to this function, and then we cannot change them or pass new ones without closing and reopening the popup, which would cause the loss of internal component's data.

A custom component would be an unjustified copypasting of all three components modified in this pr

Reply:
I think it is a bad idea to show some dialog over a popup dialog, since it is completely unexpected by user. It more looks like inplace user list with selection, and in this case you could on click show something. So please just create a new component for your needs.

Reply:
Done